hls4ml/utils/config.py

```diff
@@ -79,8 +79,8 @@ def _get_precision_from_quantizer(quantizer):
         rnd = "AP_RND_CONV"
         overflow = "AP_SAT"
         if quantizer['class_name'] in ('quantized_relu', 'quantized_relu_po2'):
-            signed = False
-            integer -= 1
+            signed = True
```

Comment:
Why do you think this should be signed?

Reply:
It should only be signed for leaky, not for the others.

Reply:
Implemented an `if` statement which makes the activation signed only if the negative slope is different from 0.